internal/mesh/internal/controllers/gatewayproxy/controller.go

```diff
@@ -104,7 +112,25 @@ func (r *reconciler) Reconcile(ctx context.Context, rt controller.Runtime, req c
 		rt.Logger.Trace("proxy state template for this gateway doesn't yet exist; generating a new one")
 	}
 
-	newPST := builder.NewProxyStateTemplateBuilder(workload).Build()
+	exportedServicesID := &pbresource.ID{
+		Name:    "global",
```

Comment:
There's a constant for this that would be nice to use so it's more self-documenting

https://github.com/hashicorp/consul/blob/0295b959c9eb22bb3a0a4df6a1957e0b2a26ff84/internal/multicluster/internal/types/computed_exported_services.go#L14

Reply:
so we can't actually import that type because it's in another `internal` package that is not a part of this one